librz/analysis/cond.c

```diff
@@ -5,22 +5,22 @@
 
 RZ_API const char *rz_analysis_cond_tostring(int cc) {
```

Comment:
Then I'd say that this should go to type.c or somewhere in librz/type.

Reply:
Agree, this should be in the `librz/type` as well.